src/app/tests/suites/TestDishwasherAlarm.yaml

```diff
@@ -0,0 +1,159 @@
+    #FeatureMap 1
+    - label: "Read Supported Attribute"
+      command: "readAttribute"
+      attribute: "Supported"
+      response:
+          value: 11
+
+    - label: "Read Mask Attribute"
+      command: "readAttribute"
+      attribute: "Mask"
+      response:
+          value: 9
+
+    - label: "Read Latch Attribute"
+      command: "readAttribute"
+      attribute: "Latch"
+      response:
+          value: 9
+
+    - label: "Read State Attribute"
+      command: "readAttribute"
+      attribute: "State"
+      response:
+          value: 9
+
+    - label: "Read feature map Attribute"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x1]
+
+    #After Modify Enabled Alarm (Valid)
+    #Support 00 1011  (11)
+    #Mask    00 0011  (3)
+    #State   00 0001  (1)
+    #Latch   00 1001  (9)
+    - label: "Modify enabled alarms Command"
+      command: "ModifyEnabledAlarms"
+      arguments:
+          values:
+              - name: "Mask"
+                value: 3
+
+    - label: "Read Mask Attribute"
+      command: "readAttribute"
+      attribute: "Mask"
+      response:
+          value: 3
+
+    - label: "Read State Attribute"
+      command: "readAttribute"
+      attribute: "State"
+      response:
+          value: 1
+
+    - label: "Read Latch Attribute"
+      command: "readAttribute"
+      attribute: "Latch"
+      response:
+          value: 9
+
+    #After Modify Enabled Alarm (Invalid)
+    #Support 00 1011  (11)
+    #Mask    00 0011  (3)
+    #State   00 0001  (1)
+    #Latch   00 1001  (9)
+
+    - label: "Modify enabled alarms Command"
+      command: "ModifyEnabledAlarms"
+      arguments:
+          values:
+              - name: "Mask"
+                value: 16
+      response:
+          error: INVALID_COMMAND
+
+    - label: "Read Mask Attribute"
+      command: "readAttribute"
+      attribute: "Mask"
+      response:
+          value: 3
+
+    - label: "Read State Attribute"
+      command: "readAttribute"
+      attribute: "State"
+      response:
+          value: 1
+
+    - label: "Read Latch Attribute"
+      command: "readAttribute"
+      attribute: "Latch"
+      response:
+          value: 9
+
+    #After Reset alarms bit 0
+    #Support 00 1011  (11)
+    #Mask    00 0011  (3)
+    #State   00 0000  (0)
+    #Latch   00 1000  (8)
+    - label: "Reset Command"
+      command: "Reset"
+      arguments:
+          values:
+              - name: "Alarms"
+                value: 1
+
+    - label: "Read Mask Attribute"
+      command: "readAttribute"
+      attribute: "Mask"
+      response:
+          value: 3
+
+    - label: "Read State Attribute"
+      command: "readAttribute"
+      attribute: "State"
+      response:
+          value: 0
+
+    - label: "Read Latch Attribute"
+      command: "readAttribute"
+      attribute: "Latch"
+      response:
+          value: 8
```

Comment:
Why did this value change?

Please have someone familiar with the Alarm Base cluster spec review this test.

Reply:
Agree with @bzbarsky-apple, the Latch attribute indicates the alarms that explicitly need a RESET.  It doesn't mutate on that RESET.  In this case Latch should still be 1001 (9).